prober/http.go

```diff
@@ -29,25 +31,34 @@ var (
 		},
 		[]string{"name", "status_code", "result", "url"},
 	)
+	dnsLookupDurations = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "health_http_dns_lookup_time_seconds",
+			Help:    "The response time of dns lookup",
+			Buckets: []float64{0.001, 0.005, 0.01, 0.02, 0.03, 0.05, 0.075, 0.1, 0.2, 0.5, 0.75, 1, 1.5, 2, 2.5, 3, 4, 5},
+		},
+		[]string{"name", "status_code", "result", "url"},
+	)
 )
 
 // TODO: it could be a bad practice!
 func init() {
 	prometheus.MustRegister(httpRequests)
 	prometheus.MustRegister(httpDurations)
+	prometheus.MustRegister(dnsLookupDurations)
+
 }
 
 func NewHttp(name string, url string, rps float64, timeout time.Duration, tlsSkipVerify bool, host string) HTTP {
 	client := &http.Client{
 		Timeout: timeout,
 	}
-
+	customTransport := http.DefaultTransport.(*http.Transport).Clone()
+	customTransport.DisableKeepAlives = true
```

Comment:
make this option configurable in the config file

Reply:
will be fixed in anther PR.

Reply:
ok, but it shouldn't go to production without this change.